ecs-init/docker/docker.go

```diff
@@ -427,6 +432,65 @@ func getDockerPluginDirBinds() []string {
 	return pluginBinds
 }
 
+// take pathPredicate as an argument for unit testing
+func getCapabilityExecBinds(pathPredicate func(path string, predicate func(fileInfo os.FileInfo) bool) (bool, error)) []string {
+	const (
+		capabilityExecName = "exec"
+
+		hostCapabilitiesResourcesRootDir      = "/home/ec2-user/capabilities-deps"
```

Comment:
this seems different from what we used in agent code. We don't need a separate dir for capabilities deps, since it would be a duplicate of the binaries we already add in the AMIs.

Reply:
didn't notice that we already defined `/home/ec2-user/ssm-agent/linux_amd64` earlier, thanks for pointing it out!

Reply:
side note on this, I think what we should do is to have a directory for capabilities' dependencies (maybe `ecs-capabilities-deps`, etc.?); otherwise, dropping `ssm-agent` on the instance's home path seems out of nowhere

Reply:
The paths used in the agent code (i.e. /home/ec2-user/ssm-agent/linux_amd64) are arbitrary and were temporary for testing. I think this is where we should define the ideal path where the resources should live. 
How are we going to place these resources into the ami? I would prefer to use one of the directories we're using for toher things like the boltdb. 

I propose:
`/var/lib/ecs/capabilities/exec`

What do you think?

Reply:
why not just `/var/lib/ecs/exec` , why do we need capabilities?